### PR TITLE
[Agreements] WHTA.Tamara version 2023.0.8.9

### PR DESCRIPTION
**Pull request by lars-nvhgroup**
cc @denelon @stephengillie

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

eula_1

### CODE REVIEW

**Comment by lars-whta** [comment]
> **Found duplicate pull request(s)**
> 
> * **Manifest Path: manifests\w\WHTA\Tamara\2023.0.8.9**
>   
>   * [[Agreements] Add back WHTA.Tamara #139463](https://github.com/microsoft/winget-pkgs/pull/139463)

Previous PR should be closed--not sanctioned.

File: manifests/w/WHTA/Tamara/2023.0.8.9/WHTA.Tamara.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+PackageVersion: 2023.0.8.9
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: -mode silent
+  SilentWithProgress: -mode silent
+  Custom: -mode silent -agreeToLicense yes
```
Comment (mdanish-kh) [inline manifests/w/WHTA/Tamara/2023.0.8.9/WHTA.Tamara.installer.yaml:14]:
```suggestion
  Custom: -agreeToLicense yes
```
Custom represents the args that are passed in addition to the Silent / SilentWithProgress args.

This means with the current manifest, the following args will be passed:

`./installer.exe -mode silent -mode silent -agreeToLicense yes`

We can remove `-mode silent` from `Custom:` to get rid of the duplication

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
Looks good now, agreements will have to be reviewed by an admin

File: manifests/w/WHTA/Tamara/2023.0.8.9/WHTA.Tamara.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+RequireExplicitUpgrade: true
+ElevationRequirement: elevationRequired
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/tekniskakustik/tamara/releases/download/v2023.0.8.9/installer.exe
+  InstallerSha256: DC10DC695C58672354E687D81E1F8950E7D9A4F35453BFD74C33A8A4F824A25A
+  AppsAndFeaturesEntries:
+  - DisplayName: Tamara
+    Publisher: Wolkesson & Hansson Teknisk Akustik AB
```
Comment (mdanish-kh) [inline manifests/w/WHTA/Tamara/2023.0.8.9/WHTA.Tamara.installer.yaml:34]:
```suggestion
    Publisher:
    ProductCode: Tamara
```

Missed this earlier, the installer doesn't write a Publisher value to the registry / Apps&Features / Control Panel. We can use ProductCode to have better package matching

image

**Review by denelon — APPROVED** [review]

**Pull request merged**